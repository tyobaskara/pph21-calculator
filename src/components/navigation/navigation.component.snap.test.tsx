import React from "react";
import renderer from 'react-test-renderer';
import { BrowserRouter } from "react-router-dom";
import NavigationComponent from './navigation.component';

test('NavigationComponent snaptest', () => {
  const component = renderer.create(
    <BrowserRouter>
      <NavigationComponent/>
    </BrowserRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});