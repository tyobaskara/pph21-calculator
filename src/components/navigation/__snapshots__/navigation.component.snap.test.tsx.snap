// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationComponent snaptest 1`] = `
<header
  className="App-header"
>
  <nav
    className="container"
  >
    <ul>
      <li
        className=""
      >
        <a
          href="/"
          onClick={[Function]}
        >
          PPH-21
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
